Engine/Engine.cpp

```diff
@@ -435,8 +422,6 @@ bool Engine::_44EEA7() {  // cursor picking - particle update
     lightmap_builder->MobileLightsCount = 0;
     decal_builder->DecalsCount = 0;
     decal_builder->curent_decal_id = 0;
-    if (!_44F07B())
```

Comment:
sorry I'm unsure about the context here really, but did you not need to replace this? _44F07B was something to do with a key being held

Reply:
it was basically checking shift and f11 keys before drawing debug info for blv - for now i think we'd like to show it anyways